lib/cucumber/runtime.rb

```diff
@@ -30,8 +30,12 @@ def initialize(original_exception, path)
   class FileNotFoundException < FileException
   end
 
-  class FeatureFolderNotFoundException < FileException
-    include FixRuby21Bug9285 if Cucumber::RUBY_2_1 || Cucumber::RUBY_2_2 || Cucumber::RUBY_2_3
+  class FeatureFolderNotFoundException < Exception
+    attr :path
+    def initialize(path)
+      super('No such file or directory - features')
+      @path = path
+    end
   end
```

Comment:
@nodo the failing test is `features/docs/getting_started.feature:7 # Scenario: Run Cucumber in an empty directory` (#1071)